### ISSUE TITLE
postinstall : patch-package throwing Error

### ISSUE THREAD

**Issue by CPLepage** [mongo-express/mongo-express#619]
Breaks the CI on `npm install`<br/>
Output:
```
patch-package 6.2.2
Applying patches...
Error: Patch file found for package mongodb-query-parser which is not present at node_modules/mongodb-query-parser
```

**Comment by rohit-gohri** [comment]
We should probably pin the dependency, `mongodb-query-parser`, to the exact version if shipping patch-package with the module

**Comment by rtritto** [comment]
Fixed with #907
